pkg/render/common.go

```diff
@@ -131,6 +131,18 @@ func setServerAuth(x *x509.Certificate) error {
 	return nil
 }
 
+func CreateOperatorTLSSecret(
```

Comment:
I created wrapped exported functions to minimise the number of changes for this PR. I can, of course, just change the unexport function to an exported one but then there will be a lot more files to review.

Reply:
Appreciate this for the first round of review :+1: 

But longer term I don't see a reason to keep them separate functions - do you?